pkg/elasticetcd/config.go

```diff
@@ -9,12 +9,20 @@ import (
 
 // Default values for the various ElasticEtcd config options
 const (
-	DefaultCURL      = "http://0.0.0.0:2379"
-	DefaultPURL      = "http://0.0.0.0:2380"
-	DefaultEndpoint  = "http://localhost:2379"
-	DefaultName      = "elasticetcd"
-	DefaultIdealSize = 3
-	DefaultDir       = "."
+	DefaultCURL          = "http://0.0.0.0:2379"
+	DefaultPURL          = "http://0.0.0.0:2380"
+	DefaultEndpoint      = "http://localhost:2379"
+	DefaultName          = "elasticetcd"
+	DefaultIdealSize     = 3
+	DefaultDir           = "."
+	DefaultCertFile      = "/var/lib/gd2/certificates/server.crt"
```

Comment:
The elasticetcd package should not contain defaults referencing GD2. It is expected to remain a standalone package, without any ties to the rest of GD2. Please move these defaults to GD2, preferrably to the store package for now.

Also, we shouldn't hardcode paths, as GD2 can use different working/config/state directories. Instead, I'd suggest setting a default directory name for a certs directory, and default filenames for the certs. Then you could build the paths based on the configured directories when you need to pass the paths. For eg.
```go
const (
  DefaultCertDir = "certificates"
  DefaultCertFile = "server.crt"
)

serverCert := path.Join(config.GetString("localstatedir", DefaultCertDir, DefaultCertFile)
```

Reply:
Agree with dirname and filename thing. Will change accordingly.

Reply:
Now its removed from EE config and whatever value is set in config file would be considered as full path if the certificates. User can choose to keep the certs in any path.